installer/deb/create_deb.sh

```diff
@@ -77,22 +77,26 @@ main() {
     mkdir -p $BUILD_DIR/usr/share/pixmaps || fatal "failed to create $BUILD_DIR/usr/share/pixmaps"
     cp $TMP_DIR/usr/local/share/pixmaps/MassaStation.png $BUILD_DIR/usr/share/pixmaps || fatal "failed to copy MassaStation.png to $BUILD_DIR/usr/share/pixmaps"
 
+    mkdir -p $BUILD_DIR/usr/share/doc/massastation || fatal "failed to create $BUILD_DIR/usr/share/doc/massastation"
```

Comment:
because you do computation in the fatal string, is there a risk of a panic without any error message? Imagine you make a mistake in `fatal "failed to copy MassaStation.png to $BUILD_DIR/usr/share/pixmaps"`, for example the variable BUILD_DIR doesn't exist, what would be the error? Maybe it's safer to put only a simple string as `fatal` argument

Reply:
In shell scripting, if a variable does not exists, it is replaced by nothing and no error is raised
So with you example, we would endup with `failed to copy MassaStation.png to /usr/share/pixmaps`
But also, `$BUILD_DIR` is set as a global variable at the beginning of the script, so that should never happen

And, anyway, this script is for building the deb package so if there were small errors like that, it should be fine

Reply:
ok!